pymc/tests/test_distributions_moments.py

```diff
@@ -751,6 +752,51 @@ def test_categorical_moment(p, size, expected):
     assert_moment_is_expected(model, expected)
 
 
+@pytest.mark.parametrize(
```

Comment:
Apologies to keep requesting changes. I think we can now drop some of the variations with `np.ones`, as they don't provide new checks for the ones with different `mu`s. Just to keep the test runtimes a bit more in check

Reply:
No problem. I have removed and changed a few test cases.